docs/topics/api/abuse.rst

```diff
@@ -0,0 +1,59 @@
+=============
+Abuse Reports
+=============
+
+The following API endpoint covers abuse reporting
+
+---------------------------------
+Submitting an add-on abuse report
+---------------------------------
+
+.. _`addonabusereport-create`:
+
+The following API endpoint allows an abuse report to be submitted for an Add-on,
+either listed on https://addons.mozilla.org or not.
+Authentication is not required, but is recommended so reports can be responded
+to if nessecary.
+
+.. http:post:: /api/v3/abuse/reportaddon/
+
+    .. _addonabusereport-create-request:
+
+    :<json string addon: The add-on id, slug, or guid of the add-on (required).
```

Comment:
"add-on" is in this sentence twice. Maybe just "The id, slug, or guid of the add-on to report for abuse."